custom/icds_reports/models/views.py

```diff
@@ -1079,3 +1079,26 @@ class Meta(object):
         app_label = 'icds_reports'
         managed = False
         db_table = 'icds_disha_indicators'
+
+
+class DishaIndicatorViewV2(models.Model):
```

Comment:
I know you have followed the model of existing DISHA AP mostly, but given this is much simpler query (only at state level and involves only two agg tables), do we actually need a database view and a SQL join. Is it not possible to directly query the agg tables?

Reply:
I believe that be possible. But, This was it is just simpler and easy to debug and maintain. Otherwise, direct querying would be difficult as two joins are not just normal they also use inner query.